docs/openapi.yaml

```diff
@@ -14,30 +14,59 @@ info:
 
     It is comprised of two parts; the Stacks Blockchain API and the Stacks Core API.
 
-    <a href="collection.json" download="collection.json">Download Postman collection</a>
+    <a href="https://hirosystems.github.io/stacks-blockchain-api/collection.json" download="collection.json">Download Postman collection</a>
+
+tags:
+  - name: Faucets
+    description: |
+      Faucets are testnet only endpoints for delivering testnet tokens to a specified address. These are useful for
+      obtaining tokens to test transactions, exercise smart contracts, or perform network operations such as stacking or
+      mining on the testnet.
 
 paths:
   /extended/v1/faucets/stx:
     parameters:
       - name: address
         in: query
-        description: STX address
+        description: A valid testnet STX address
```

Comment:
we could point out that these addresses have to start with `ST`

Reply:
This field is meant to hold a very short string, it isn't meant for a long description. I can add that information in the description of the endpoint, however.